src/lib/integration_test_lib/network_state.ml

```diff
@@ -82,6 +85,37 @@ module Make
                        ~data:new_best_tip
                    in
                    {state with best_tips_by_node= best_tips_by_node'} ) ) )) ;
+    let handle_gossip_received event_type =
+      ignore
+        (Event_router.on event_router event_type
+           ~f:(fun node gossip_with_direction ->
+             update ~f:(fun state ->
+                 { state with
+                   gossip_received=
+                     Map.update state.gossip_received (Node.id node)
+                       ~f:(fun gossip_state_opt ->
+                         let gossip_state =
+                           match gossip_state_opt with
+                           | None ->
+                               Gossip_state.create (Node.id node)
+                           | Some state ->
+                               state
+                         in
+                         [%log debug] "GOSSIP RECEIVED by $node"
```

Comment:
Remove?

Reply:
i think it's a useful debug log isn't it?  certainly helped me debug this.  will remove if you feel strongly tho